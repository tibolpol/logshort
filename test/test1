#!/bin/bash

mydir="$(dirname "$0")"
"$mydir/../src/logshort" < "$mydir/test1.in" | diff - "$mydir/test1.out" &&
echo $0 OK || echo $0 KO
