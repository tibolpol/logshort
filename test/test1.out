aij_read - rc=1 - dsn q sqdre
opajdeeimport - rc=1
aij_read - rc=1 - dsn q sqdre
==== 43 consecutive 100% matching 2*lines ====
gpg: Signature faite le lun. 24 sept. 2023 10:17:43 CET
gpg:                avec la clef RSA A2FCE7347BBC1BBA50F3F8E4B5735B7EDF46595D
gpg: Bonne signature de « Duschmol (with no passphrase) <duschmol@foo.bar> » [ultime]
gpg: Signature faite le lun. 24 sept. 2023 10:17:43 CET
gpg:                avec la clef RSA A2FCE7347BBC1BBA50F3F8E4B5735B7EDF46595D
==== 25 consecutive 93% matching 3*lines ====
2023-09-24 10:28:31 test1@tlpbuster - aij_read(+10) bnachecked : 104395 sqdgnes - md5sum:c8f480a079b00ddbad5652da8917ef16, aagrj aagrk aagrl aagrm aagrn aagro aagrp aagrq aagrr aagrs
aij_read - rc=0 - fin normale (aij_read -10 to undo)
Job :IMP_INTERFACE : Id = 7005892
. . . . . .COMPLETED_ERROR
24/09/2023 10:28:33 [INFO ][by sopoizez] - # Début Import : IMP_INTERFACE_7005892.txt (stdin.48f).
 24/09/2023 10:35:11 [ERROR][by sopoizez] - [import sequence:ftp://***@kjhlkhj/DCICLI/COMPTE/COMPTE23-09-20_308.TXT:1] - ERR_LKIHJZ_OBJ:Account: Référence [LKJAI - 61202205] lhjistlhjo. (F_LMakoingVlkhjdror.java:429)
==== 3 consecutive 86% matching 2*lines ====
 24/09/2023 10:35:11 [ERROR][by sopoizez] - [import sequence:ftp://***@kjhlkhj/DCICLI/COABON/COABON23-09-20_923.TXT:2] - ERR_LKIHJZ_OBJ:CsqdentUser: Référence [CIDE - 197VV235] lhjistlhjo. (F_LMakoingVlkhjdror.java:614)
 24/09/2023 10:35:11 [ERROR][by sopoizez] - [import sequence:ftp://***@kjhlkhj/DCICLI/COABON/COABON23-09-20_923.TXT:3] - ERR_LKIHJZ_OBJ:CsqdentUser: Référence [CIDE - 74722856] lhjistlhjo. (F_LMakoingVlkhjdror.java:614)
 24/09/2023 10:35:11 [ERROR][by sopoizez] - [import sequence:ftp://***@kjhlkhj/DCICLI/AUTEDI/AUTEDI23-09-20_557.TXT:1] - ERR_LKIHJZ_OBJ:Auloihjoinizasdf EDI: Référence [LKJAI - 94605538] lhjistlhjo. (F_LMakoingVlkhjdror.java:1000)
==== 5 consecutive 93% matching 1*lines ====
 24/09/2023 10:35:11 [ERROR][by sopoizez] - [import sequence:ftp://***@kjhlkhj/DCICLI/AUTKJA/AUTKJA23-09-20_645.TXT:1] - ERR_LKIHJZ_OBJ:Auloihjoinizasdf: Référence [LKJAI - 94605538] lhjistlhjo. (F_LMakoingVlkhjdror.java:774)
==== 56 consecutive 93% matching 1*lines ====
get : share jok/proxy-kjhlkhj-21 serving key "KJA/CHARCO/CHARCO23-08-28_393.TXT"
get : rc=0 -
get : share jok/proxy-kjhlkhj-21 serving key "KJA/CLIENT/SI_CLIENT_28_08_2023_0862.TXT"
get : rc=0 -
==== 2 consecutive 86% matching 2*lines ====
get : share jok/proxy-kjhlkhj-21 serving key "KJA/COABON/COABON23-08-28_562.TXT"
get : rc=0 -
get : share jok/proxy-kjhlkhj-21 serving key "KJA/AUTKJA/AUTKJA23-08-28_243.TXT"
get : rc=0 -
get : share jok/proxy-kjhlkhj-21 serving key "KJA/AUTEDI/AUTEDI23-08-28_390.TXT"
get : rc=0 -
get : share jok/proxy-kjhlkhj-21 serving key "KJA/AUTSMS/AUTSMS23-08-28_869.TXT"
get : rc=0 -
get : share jok/proxy-kjhlkhj-21 serving key "KJA/SDFTES/SDFTES23_08_28_648.TXT"
get : rc=0 -
==== 2 consecutive 93% matching 2*lines ====
get : share jok/proxy-kjhlkhj-21 serving key "KJA/PROFIL/PROFIL23-08-28_627.TXT"
get : rc=0 -
get : share jok/proxy-kjhlkhj-21 serving key "KJA/AQKCPT/AQKCPT23-08-28_741.TXT"
====={ K__A4:ftp://***@kjhlkhj/KJA/AQKCPT/AQKCPT23-08-28_741.TXT:14148
IBAN=AP  111000000197956
CCUR=KHJ
NDEC=2
FDAT=Mon Aug 27 00:00:00 CET 2023
FSGN=+
FSOL=20000
XCUR=KHJ
=====}
====={ K__A4:ftp://***@kjhlkhj/KJA/AQKCPT/AQKCPT23-08-28_741.TXT:14154
IBAN=AP  111000000199702
CCUR=APD
NDEC=2
FDAT=Mon Aug 27 00:00:00 CET 2023
FSGN=+
FSOL=1000000
XCUR=APD
==== 150 consecutive 85% matching 9*lines ====
FSGN=-
==== 19 consecutive 85% matching 9*lines ====
XCUR=KHJ
=====}
====={ K__A4:ftp://***@kjhlkhj/KJA/AQKCPT/AQKCPT23-08-28_741.TXT:57249
IBAN=AP  156000000075523
CCUR=APD
NDEC=2
FDAT=Mon Aug 27 00:00:00 CET 2023
==== 63 consecutive 85% matching 9*lines ====
FSGN=-
==== 55 consecutive 85% matching 9*lines ====
XCUR=KHJ
==== 222 consecutive 85% matching 9*lines ====
FSGN=-
FSOL=4748
XCUR=APD
=====}
====={ K__A4:ftp://***@kjhlkhj/KJA/AQKCPT/AQKCPT23-08-28_741.TXT:142534
IBAN=AP  374000000094596
==== 90 consecutive 85% matching 9*lines ====
CCUR=KHJ
==== 29 consecutive 85% matching 9*lines ====
FSGN=-
FSOL=258129
XCUR=APD
=====}
====={ K__A4:ftp://***@kjhlkhj/KJA/AQKCPT/AQKCPT23-08-28_741.TXT:171065
IBAN=AP  433000000075447
CCUR=KHJ
NDEC=2
FDAT=Mon Aug 27 00:00:00 CET 2023
FSGN=+
FSOL=13699
XCUR=KHJ
=====}
====={ K__A4:ftp://***@kjhlkhj/KJA/AQKCPT/AQKCPT23-08-28_741.TXT:172641
IBAN=AP  434223609151088
CCUR=APD
NDEC=2
FDAT=Mon Aug 27 00:00:00 CET 2023
==== 8 consecutive 85% matching 9*lines ====
